Documentation/Update Snapshot Used by Droplet Deployer.md

```diff
@@ -7,7 +7,7 @@
 1. Replicate [the snapshot][0] to all regions (click the "More" option, then "Add to Region").
 1. Rename the [old snapshot][0] to `Old Droplet Deployer` (check "Created" values).
 1. [Generate a new Personal Access Token][2].
-1. To get the ID of the newly-created snapshot, run `curl -sX GET -H "Content-Type: application/json" -H "Authorization: Bearer <token here>" "https://api.digitalocean.com/v2/images?private=true" | sed -n 's/.*"id":\([^,]*\),"name":"Droplet Deployer".*/\n\1\n\n/p'`
+1. To get the ID of the newly-created snapshot, run `curl -sX GET -H "Content-Type: application/json" -H "Authorization: Bearer <token here>" "https://api.digitalocean.com/v2/images?private=true" | sed -n 's/.*"id":\([^,]*\),"name":"Droplet Deployer".*/\1/p'`
```

Comment:
This makes the output awkward on a Linux terminal; with no trailing EOL the ID number is output and the next terminal prompt is appended.  To use your same example, the output is `1234fraser@vault0: `.

We can keep the original intended behaviour by using `printf`:
```
printf "\n%d\n\n" $(curl -sX GET -H "Content-Type: application/json" -H "Authorization: Bearer <token here>" "https://api.digitalocean.com/v2/images?private=true" | sed -n 's/.*"id":\([^,]*\),"name":"Droplet Deployer".*/\1/p')
```